src/patternfly/components/Button/button.hbs

```diff
@@ -1,7 +1,10 @@
 <button class="pf-c-button {{ btnClass }}" 
   {{#if btnAttributes}}
     {{{ btnAttributes }}} 
-  {{/if}} 
+  {{/if}}
+  {{#if aria-label}}
+    aria-label="{{aria-label}}"
+  {{/if}}
```

Comment:
Instead of adding this here, we _can_ just use `btnAttributes` defined above. Here's an example from the [Button component](https://github.com/patternfly/patternfly-next/blob/master/src/patternfly/components/Button/examples/buttonTypes.hbs#L17):
`{{#> button btnClass="pf-m-action" btnAttributes="aria-label=\"Remove\" "}}`

Maybe this is something that we should standardize on, as far as when we add handlebar properties for specific attributes, like `class`, `id`, and whether it's expanded, vs having a catch all property for defining any random attribute. I kind of lean toward not including handlebar properties for specific attributes unless there's a benefit in doing so (e.g. with `id` or `is-expanded`, that property can be used in different places throughout the component). @matthewcarleton, @michael-coker, @srambach, do you have any thoughts on this one?

Reply:
@jgiardino agreed.

Reply:
- I would advocate for namespacing based on the component name, rather than the associated class (as you suggested). Also, we can wrap `component--attributes` in single quotes rather than escaping each double quote. 
- Any inherited attribute could be set explicitly, then called where needed in child components. 
```
<!-- block.hbs component definition -->
<div class="pf-c-block{{#if block--modifiers}} {{block--modifiers}}{{/if}}" 
  {{#if block--attributes}}
    {{{block--attributes}}}
  {{/if}}
  {{#if block--is-expanded}}
    is-expanded="true"
  {{/if}}
>
  {{> @partial-block }}
</div>
```
```
<!-- block-item.hbs component definition -->
<div class="pf-c-block__item{{#if block-item--modifiers}} {{block-item--modifiers}}{{/if}}{{#if block--is-expanded}} pf-m-expanded{{/if}}" 
  {{#if block-item--attributes}}
    {{{block-item--attributes}}}
  {{/if}}
>
  {{> @partial-block }}
</div>
```
The when calling component, use single quotes for wrapping, like:
```
{{#> block block--modifiers="pf-m-modifier" block--attributes='aria-label="Close Dialog" another-attribute="modal-close"' block--is-expanded="true"}}
  {{#> block-item block-item--modifiers="pf-m-modifier" block-item--attributes='aria-label="something" another-attribute="something else"'}}
    some content
  {{/block-item}}
{{/block}}
```